### PR TITLE
m6 release pipeline

### PR DESCRIPTION
**Pull request by scoheb**
- include disabling cosign experimental features

### CODE REVIEW

File: definitions/m6-release-pipeline/m6-release-pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        - name: applicationSnapshot
+          value: $(tasks.apply-mapping.results.applicationSnapshot)
+    - name: verify-enterprise-contract
+      taskRef:
+        name: verify-enterprise-contract-v2
+        bundle: quay.io/scoheb/verify-enterprise-contract-v2:v2
+      params:
+        - name: IMAGES
+          value: $(tasks.apply-mapping.results.applicationSnapshot)
+#        - name: REKOR_HOST
```
Comment (davidmogar) [inline definitions/m6-release-pipeline/m6-release-pipeline.yaml:81]: Could you remove these comments?

File: definitions/m6-release-pipeline/m6-release-pipeline.yaml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+      taskRef:
+        name: prepare-validation
+        bundle: quay.io/hacbs-release/prepare-validation:main
+      params:
+        - name: applicationSnapshot
+          value: $(tasks.apply-mapping.results.applicationSnapshot)
+    - name: verify-enterprise-contract
+      taskRef:
+        name: verify-enterprise-contract-v2
+        bundle: quay.io/scoheb/verify-enterprise-contract-v2:v2
```
Comment (davidmogar) [inline definitions/m6-release-pipeline/m6-release-pipeline.yaml:77]: Maybe better to push it to [hacbs-release-tests](quay.io/organization/hacbs-release-tests) or even [hacbs-release](quay.io/organization/hacbs-release).

Comment (scoheb) [reply definitions/m6-release-pipeline/m6-release-pipeline.yaml]: updated to be similar to other bundle references in hacbs-release

**Review by davidmogar — APPROVED** [review]

**Comment by scoheb** [comment]
@davidmogar I removed the prepare step since it is no longer needed since we are using the snapshot directly now in M6

**Pull request merged**